docs/docs/concepts/email-sms.md

```diff
@@ -92,12 +92,70 @@ courier:
   #    > set COURIER_TEMPLATE_OVERRIDE_PATH=<value>
   #
   template_override_path: /conf/courier-templates
+
+  ## Override with remote templates ##
+  #
+  # You can specify specific template values to override or the whole template
+  #
+  # Supported templates are:
+  # - verification
+  #   - valid
+  #   - invalid
+  # - recovery
+  #   - valid
+  #   - invalid
+  #
+  # Each template supports the following layout. A singular key can be specified under `email` to override the defaults.
```

Comment:
SO this means specify e.g. `subject` does not require us to set `html`?

Reply:
yes, they are independent